style.css

```diff
@@ -0,0 +1,466 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+html {
+  background-color: lightgray;
+}
+
+body {
+  width: 375px;
```

Comment:
- [ ] One of the key requirements of this project is to have a responsive design. giving exact widths to containers make it impossible to have a responsive design. In this milestone, your design should be responsive on all screens from `375px` to `768px`. 
Kindly consider removing this line and then adjust your design to have responsiveness.
your design at 709px wide screen
![qw](https://user-images.githubusercontent.com/96683843/235980992-5f7ca643-ffe3-4b20-9496-5589b1caf242.jpg)

Reply:
yep, remove the width but obviously after removing it, you will need to add some stylings to keep your design responsive.